crates/config/src/zksync.rs

```diff
@@ -126,8 +127,9 @@ impl ZkSyncConfig {
                     per_contract: Some([OutputSelectionFlag::ABI].into()),
                 }),
             },
-            cli_settings: CliSettings::default(),
-        }
+        };
+
+        ZkSolcSettings { settings: zk_settings, cli_settings: CliSettings::default() }
```

Comment:
```suggestion
         // `cli_settings` get set from `Project` values when building `ZkSolcVersionedInput` 
        ZkSolcSettings { settings: zk_settings, cli_settings: CliSettings::default() }
```
Maybe add a comment on why we use the default values here as it might get confusing to follow when these settings are actually set